src/components/SidebarButton/SidebarButton.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import MenuIcon from '@mui/icons-material/Menu';
+import LogoutIcon from '@mui/icons-material/Logout';
+import Popper from '@mui/material/Popper';
+import Fade from '@mui/material/Fade';
+import { Avatar } from '@/components/Avatar';
+import { Seperator } from '@/components/Seperator';
+
+const SidebarButton = ({ src, nickname }) => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const [open, setOpen] = React.useState(false);
+  const [placement, setPlacement] = React.useState();
+
+  const handleClick = (newPlacement) => (event) => {
```

Comment:
이후부터는 evernt를 사용할때는 인수부터 구조 분해 할당해서 꺼내 쓰는 걸로 할게요~!

```suggestion
  const handleClick = (newPlacement) => ({currentTarget}) => {
```